src/scss/components/_button.scss

```diff
@@ -43,6 +45,7 @@
   height: rhythm(9);
   justify-content: center;
   margin: 0 20px;
+  padding: 20px;
```

Comment:
> I added a little padding to the social issues. Did that fix the problem with them looking like ovals instead of circles?

It does. What solved the problem is that you overrode `.button`'s `padding: 13px 37px` with this `20px`. So they're now round because the padding is even (and explicit `height` and `width` dimensions are defined). 👍